LibreNMS/Alert/RunAlerts.php

```diff
@@ -188,7 +188,7 @@ public function describeAlert($alert)
         $obj['uid'] = $alert['id'];
         $obj['alert_id'] = $alert['alert_id'];
         $obj['severity'] = $alert['severity'];
-        $obj['rule'] = $alert['rule'];
+        $obj['rule'] = $alert['rule'] ?: json_encode($alert['builder']);
```

Comment:
do we keep $obj['builder'] = $alert['builder']; line 187 then ?

Reply:
@PipoCanaja yes, that is the array of the builder rule... not json.  Other option is to remove the "rule" field completely, but since we haven't removed legacy rules (especially from the collection).  It doesn't make much sense yet.